pkg/logging/server.go

```diff
@@ -45,10 +45,39 @@ func (s Service) Register(r *grpc.Server) {
 type Server struct {
 }
 
-// SetDebug enable/disable debug mode for logging package
-func (s *Server) SetDebug(ctx context.Context, req *logging.SetDebugModeRequest) (*logging.SetDebugModeResponse, error) {
-	return &logging.SetDebugModeResponse{
-		ResponseStatus: logging.ResponseStatus_OK,
+// GetLevel implements GetLevel rpc function to get a logger level
+func (s *Server) GetLevel(ctx context.Context, req *logging.GetLevelRequest) (*logging.GetLevelResponse, error) {
+
+	name := req.GetLoggerName()
+	if name == "" {
+		return &logging.GetLevelResponse{}, errors.New("precondition for get level request is failed")
+	}
+
+	names := strings.Split(name, "/")
```

Comment:
I think we have a constant defined for `"/"`... `nameSep`

Reply:
We could also move this `Split` call to a function.

Reply:
OK. Let me fix it.

Reply:
Done